conf.d/ucmdb_file.yaml.example

```diff
@@ -0,0 +1,23 @@
+# UCMDB file base integration
+
+init_config:
+
+instances:
+  # File location where tql query result is synced from
+  - location: "/path/to/file"
```

Comment:
We should have some polling interval to resend the data periodically (this will make sure data comes in after a check restart). We should also immediately resend the data on a timestamp change